.github/dependabot.yml

```diff
@@ -142,6 +143,27 @@ updates:
       - dependency-name: org.eclipse.microprofile.openapi:microprofile-openapi-tck
       - dependency-name: org.eclipse.microprofile.opentracing:microprofile-opentracing-tck*
       - dependency-name: org.eclipse.microprofile.rest.client:microprofile-rest-client-tck
+      # Others
+      - dependency-name: com.puppycrawl.tools:checkstyle
+      - dependency-name: com.google.cloud.functions:*
+      - dependency-name: com.google.errorprone:*
+      - dependency-name: com.google.http-client:*
+      - dependency-name: com.nimbusds:nimbus-jose-jwt
+      - dependency-name: io.apicurio:*
+      - dependency-name: io.dekorate:servicebinding-annotations
+      - dependency-name: io.opentelemetry:*
+      - dependency-name: io.opentelemetry.*:*
+      - dependency-name: io.opentracing.contrib:*
+      - dependency-name: org.aesh:readline
+      - dependency-name: org.checkerframework:checker-qual
+      - dependency-name: org.jboss.metadata:jboss-metadata-web
+      - dependency-name: org.jboss.modules:jboss-modules
+      - dependency-name: com.unboundid:unboundid-ldapsdk
+      - dependency-name: org.commonmark:commonmark
+      - dependency-name: org.webjars:*
```

Comment:
Don't forget to remove https://github.com/quarkusio/quarkus/pull/22238/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R69

Reply:
Yeah, I kept it that way as I'm unsure about this rule so I might have to remove it. I'm not entirely sure some webjars aren't used for tests and should stay the version they are.

Reply:
So, I'll see how it goes once merged and adjust accordingly.

Reply:
BTW, I will merge it tonight after the release as there's a good chance it will trigger several updates.